app/src/main/java/org/gem/indo/dooit/views/main/MainActivity.java

```diff
@@ -64,15 +64,12 @@ protected void onCreate(Bundle savedInstanceState) {
                 @Override
                 public void onGlobalLayout() {
                     // Remember to remove it if you don't want it to fire every time
-                    MainActivity.this.toolbar.getViewTreeObserver().removeGlobalOnLayoutListener(this);
-                    getSupportActionBar().setHomeAsUpIndicator(R.drawable.ic_d_profile);
+//                    MainActivity.this.toolbar.getViewTreeObserver().removeGlobalOnLayoutListener(this);
```

Comment:
@wscheffer Please take a look at this and let me know if this is an acceptable solution.

The toolbar profile image is not loading when setting a new one from ProfileActivity and returning to the MainActivity. Keeping the event handler around reloads it, but I am unsure what the repercussions are. Should we rather do this somewhere else, like onResume?